mux/router_test.go

```diff
@@ -5,6 +5,9 @@ import (
 
 	"github.com/plgd-dev/go-coap/v3/mux"
 	"github.com/stretchr/testify/require"
+
+	// TODO: replace with standard maps package as soon as Go dependency hits 1.20
+	"golang.org/x/exp/maps"
```

Comment:
Please remove the empty line, reorder the imports if needed, and add a comment on the same line as the import to satisfy the linter.